app/main.py

```diff
@@ -1,6 +1,15 @@
 from typing import Callable
 
+new_set = {}
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def wrapper(*args, **kwargs):
+        if (func.__name__, *args) in new_set:
+            print("Getting from cache")
```

Comment:
The `wrapper` function checks if the result is in `new_set` but does not return the cached result. You should add a `return` statement to return the cached result from `new_set`.